.github/workflows/ci.yml

```
@@ -76,9 +81,16 @@
         env:
           TEST: true
       - name: Check code
-        run: ls -all
+        run: |
+          ls -all;pwd;whoami;
       - name: Set up date and time variable
-        run: echo "DATE_TIME=$(TZ='Asia/Tokyo' date '+%Y-%m-%d-%H-%M-%S')" >> $GITHUB_ENV
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:66: Double quote to prevent globbing and word splitting

(shellcheck)